astropy/modeling/fitting.py

```diff
@@ -37,13 +37,10 @@
 from .optimizers import (SLSQP, Simplex)
 from .statistic import (leastsquare)
 from .optimizers import (DEFAULT_MAXITER, DEFAULT_EPS, DEFAULT_ACC)
+from astropy.utils.compat.optional_deps import HAS_PKG_RESOURCES
```

Comment:
This one might conflict with astropy/astropy#11091 . How to resolve the conflict depends on which PR gets merged first. ⚔️  cc @saimn